webui/react/src/pages/ModelVersionDetails.tsx

```diff
@@ -229,6 +230,19 @@ const ModelVersionDetails: React.FC = () => {
     ];
   }, [modelVersion?.checkpoint]);
 
+  if (
+    modelVersion?.checkpoint.experimentConfig?.integrations?.pachyderm !== undefined &&
+    checkpointInfo.find(({ label }) => label === 'Data Input') === undefined
```

Comment:
see above re: when is this check going to return false? Furthermore, don't mutate objects outside of `useMemo` -- do this inside of the closure.